### ISSUE TITLE
🧊🔥💥 Implement normal, fast and urgent parameter settings for SEND flow

### ISSUE THREAD

**Issue by vkjr** [status-im/status-react#21019]
# Description

As a starting step, we need to give the user three options to set the parameters automatically. These options dialog will be accessible via the txn summary screen. 

NOTE: This might need a go change. If you pick this up, please confirm that Status Go has an end point to calculate various option parameters.

[Figma](https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17671-129320&t=wTHMEe7S0hwBoEwh-4). 

## Implementation

| Feature                    | Screenshot                                                                                                                                          |
|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
| Txn summary with parameter change anchor point | ![Image](https://github.com/user-attachments/assets/b6748f63-7f7e-455c-8985-d740f580e45f) |
| Parameters bottom sheet    | ![Image](https://github.com/user-attachments/assets/7d4676cd-3961-4bcd-b7c9-3b86a316c261) |
| Updated params             | ![Image](https://github.com/user-attachments/assets/be4399b5-a104-4dac-8287-ab02e4c84574) |

## Acceptance Criteria

- [ ] On the txn summary screen, there should be an option to change gas parameters
- [ ] When the anchor is clicked, a menu should open with three options (the custom option will be disabled in the context of this ticket)
- [ ] On pressing any option, the bottom sheet should close and the user should see the summary screen again
- [ ] On the summary screen, the gas price should be updated
- [ ] On the summary screen, the label should change depending on the selection : Normal, Fast, Urgent
- [ ] The UI should be hidden behind a feature flag

**Comment by vkjr** [comment]
The figma designs for this screen are [in progress](https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17669-40876&t=TK3SuJ90iMM5VoXX-1) and issue should be updated when they are finished.

**Comment by shivekkhurana** [comment]
@vkjr This issue is ready to start. There might be some changes regarding UI, but the flow is finalised and we can start building for 2.32.

**Comment by vkjr** [comment]
@shivekkhurana, thanks!

**Comment by churik** [comment]
I do not think that it is feasible to include this feature to  2.32, postponing due to limited capacity in order to keep release scope clear

cc @shivekkhurana

**Comment by vkjr** [comment]
returning the `blocked` label because at this moment designs are ready but `status-go` part is not implemented yet.
